cmd/deploy.go

```diff
@@ -52,26 +57,25 @@ var deployCmd = &cobra.Command{
 		// create root CA
 		tpl, err := template.ParseFiles(rootCaCsrTemplate)
 		if err != nil {
-			log.Fatalf("failed to parse rootCACsrTemplate: %v", err)
+			return fmt.Errorf("failed to parse rootCACsrTemplate: %v", err)
 		}
 		rootCerts, err := c.GenerateRootCa(tpl, clab.CaRootInput{Prefix: c.Conf.Prefix})
 		if err != nil {
-			log.Fatalf("failed to generate rootCa: %v", err)
-		}
-		if debug {
```

Comment:
so we want to display cert content even when debug is not on?

Reply:
The log.Debugf takes care of that. the debug level is already set in rootCmd.PreRun command.
the if debug here is redundant.